docs/data-import.rst

```diff
@@ -19,17 +21,25 @@ To load the database dump, run:
     unzip traffic_stops_nc_production_20150801.tar.zip
     pg_restore -Ox -d traffic_stops_nc traffic_stops_nc_production.tar
 
-To create the database dump, run:
+Browse https://s3-us-west-2.amazonaws.com/openpolicingdata/ to see what dumps
+are available.
+
+To create a new database dump, run:
 
 .. code-block:: bash
 
     pg_dump -Ox -Ft traffic_stops > traffic_stops.tar
 
+That can be loaded with the ``pg_restore`` command shown above.
 
 Raw NC Data (slower)
 ____________________
 
-Make sure our NC database is in the right state before importing:
+Command-line
+++++++++++++
+
+Make sure our NC database is in the right state before importing.
+This is not required each time data is imported.
 
```

Comment:
See updates.  Basically, there's nothing magic about when to create the database and apply migrations unless you've been playing around with it.